prody/proteins/interactions.py

```diff
@@ -1732,8 +1733,8 @@ def calcLigandInteractions(atoms, **kwargs):
                 pdb_name = pdb_name[:-4]+'_addH.pdb'
                 atoms = parsePDB(pdb_name)
                 LOGGER.info("Lack of hydrogens in the structure. Hydrogens have been added.")
-        except: 
-            LOGGER.info("Install Openbabel to add missing hydrogens or provide structure with hydrogens")
+        except ImportError: 
```

Comment:
this one could be an import error but there are probably other errors that could happen too. It would also be better for addHydrogens or whatever it is called now to handle the import errors itself.

Reply:
Done.